apps/aecore/src/aec_db.erl

```diff
@@ -558,13 +564,18 @@ gc_tx(TxHash) ->
            BlockHash when is_binary(BlockHash) ->
                {error, BlockHash};
            mempool ->
-               delete(aec_tx_pool, TxHash);
+               delete(aec_tx_pool, TxHash),
+               delete(aec_signed_tx, TxHash),
```

Comment:
I see this is deleting actual item. Ok.

Reply:
Shall this be update a metric? It is optional though may help understand what node is doing. Cc @dincho

Reply:
No opinion here 🙂